tests/providers/google/cloud/hooks/test_cloud_run.py

```diff
@@ -0,0 +1,79 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import pytest
+from unittest import mock
+from uuid import UUID
+
+from airflow.providers.google.cloud.hooks.cloud_run import CloudRunJobHook
+
+
+TASK_ID = "test-cloud-run-operator"
+JOB_NAME = "test-cloud-run-job"
+CLOUD_RUN_JOBS_STRING = "airflow.providers.google.cloud.hooks.cloud_run.{}"
+TEST_PROJECT = "test-project"
+MOCK_UUID = UUID("cf4a56d2-8101-4217-b027-2af6216feb48")
+MOCK_UUID_PREFIX = str(MOCK_UUID)[:8]
+EXECUTION_ID = f"{JOB_NAME}-{MOCK_UUID_PREFIX}"
+
+
+@pytest.fixture
+def hook() -> CloudRunJobHook:
+    return CloudRunJobHook(gcp_conn_id="google_cloud_default")
+
+
+@mock.patch(CLOUD_RUN_JOBS_STRING.format("_CloudRunJobExecutionController"))
+@mock.patch(CLOUD_RUN_JOBS_STRING.format("CloudRunJobHook.get_conn"))
+def test_execute_cloud_run_job(mock_conn, mock_controller, hook, subtests):
+    for wait_until_finished in [True, False]:
+        with subtests.test():
```

Comment:
```suggestion
@pytest.mark.parametrize("wait_until_finished", [True, False])
def test_execute_cloud_run_job(mock_conn, mock_controller, hook, wait_until_finished):
```

Do not forget to change indention of blocks